index.html

```diff
@@ -168,7 +168,7 @@ <h1>Multi-Post
             </div>
         </article>
     </section>
-
+    <div id="grid-container">
     <section class="about-section" id= "item-3">
         <article>
             <h1>About <br>
```

Comment:
- [ ] We are not done yet, it is said to use the same alignment as per the original design, well despite we are working on the desktop version already, I think you would want to take a few steps back to the mobile version and fix this below, from the `about me` section, I think we are missing some spaces on the left of the element in there, please adjust them and your work will be good to go, this is what I am talking about

<img width="431" alt="no" src="https://user-images.githubusercontent.com/77843050/206567350-30460a7e-f90d-41bc-b128-ac8184157036.png">

- [ ] Figma design

<img width="286" alt="fig" src="https://user-images.githubusercontent.com/77843050/206567784-e96031b7-5685-44a2-91ca-0972d9bbe40b.png">